cmd/cloudflared/tunnel/cmd.go

```diff
@@ -313,9 +314,9 @@ func TunnelCommand(c *cli.Context) error {
 	// Run a quick tunnel
 	// A unauthenticated named tunnel hosted on <random>.<quick-tunnels-service>.com
 	// We don't support running proxy-dns and a quick tunnel at the same time as the same process
-	shouldRunQuickTunnel := c.IsSet("url") || c.IsSet(ingress.HelloWorldFlag)
+	shouldRunQuickTunnel := c.IsSet("url") || c.IsSet("unix-socket") || c.IsSet(ingress.HelloWorldFlag)
```

Comment:
I haven't done any side-effect analysis to check whether anyone could be relying on `--unix-socket` falling through to `tunnel run`, but I believe it should be okay.